Octokit.Tests.Integration/Clients/IssuesClientTests.cs

```diff
@@ -230,6 +254,14 @@ public IssuesClientTests()
         Assert.True(retrieved.Any(i => i.Number == issue2.Number));
         Assert.True(retrieved.Any(i => i.Number == issue3.Number));
         Assert.True(retrieved.Any(i => i.Number == issue4.Number));
+        var issue = retrieved[0];
+        Assert.Equal(4, issue.Reactions.TotalCount);
```

Comment:
The reactions created above are one-per-issue. So we only expect `1` here, not 4. Or perhaps we can simplify the test?